### ISSUE TITLE
Xcode 7.3 update is not backwards compatible

### ISSUE THREAD

**Issue by stevesarkisian-tribpub** [PhamBaTho/BTNavigationDropdownMenu#56]
So the update you just pushed for XCode 7.3 is not backwards compatible.  I wished you'd have tagged it 0.3.1 or something.

**Comment by PhamBaTho** [comment]
Which kind of errors did you receive @stevesarkisian-tribpub? About #selector of Swift 2.2? Sorry to ask this because I don't have XCode 7.2 version in my Mac now, upgraded it to 7.3 already.

**Comment by stevesarkisian-tribpub** [comment]
Yes, it was due to the #selector.  It may have been better to create a branch for XCode 7.3.
